src/main/java/uk/gov/hmcts/darts/usermanagement/service/impl/UserManagementServiceImpl.java

```diff
@@ -99,6 +103,21 @@ public List<UserWithIdAndTimestamps> search(UserSearch userSearch) {
         return userWithIdAndLastLoginList;
     }
 
+    @Override
+    public List<UserWithIdAndTimestamps> getUsers(int courtCasedId, String emailAddress) {
+        List<UserWithIdAndTimestamps> userWithIdAndLastLoginList = new ArrayList<>();
+
+        userManagementQuery.getUser(emailAddress)
```

Comment:
I think this can be replaced with a new repository method, like below

```
userAccountRepository.findByEmailAddressIgnoreCaseAndIsSystemUser(emailAddress, false);
```

Obviously you would need to add that to the `UserAccountRepository` but it only requires the method signature. This would mean you could remove the `UserManagementQuery` stuff, unless I'm missing a different reason why it's there?